src/Infrastructure/Resources/config/app/config.yml

```
@@ -30,7 +30,7 @@ knp_gaufrette:
     adapters:
         sylius_comments_attachment:
             local:
-                directory: "%kernel.root_dir%/../web/media/comment_attachments"
+                directory: "%kernel.root_dir%/../public/media/comment_attachments"
```

We should probably use `%sylius_core.public_dir%` instead, and ultimately move it to the configuration(but this part could be done later). 
```suggestion
                directory: "%sylius_core.public_dir%/media/comment_attachments"
```